test/integration2/schema/jsonb.spec.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai');
+const { getAllDbs, getKnexForDb } = require('../util/knex-instance-provider');
+
+describe('Schema', () => {
+  describe('json columns', () => {
+    // no support for json in mssql, omit test
+    getAllDbs()
+      .filter((db) => db !== 'mssql')
```

Comment:
Does it work on Oracle or that also has to be skipped?

Reply:
Seems to be the same situation as mssql - it's not a supported data type. Attempting to create such a schema gives `Error: create table "table_with_jsonb_col1" ("jsonb_col1" undefined) - ORA-00902: invalid datatype`

Reply:
the reason I hadn't put it in the test, was because it wasn't part of `getAllDbs`, I assumed there was a reason for it not being there so I didn't explicitly add it.

Reply:
Let's skip it too then.

Reply:
I'll add it to the filter